src/test/java/teammates/test/cases/ui/InstructorFeedbackQuestionCopyActionTest.java

```diff
@@ -29,97 +29,97 @@ public void caseSetUp() throws Exception {
         dataBundle = getTypicalDataBundle();
         removeAndRestoreTypicalDataInDatastore();
     }
-    
+
     @Test
-    public void testAccessControl() throws Exception{
+    public void testAccessControl() throws Exception {
         String[] params = new String[]{
                 Const.ParamsNames.FEEDBACK_SESSION_NAME, "First feedback session",
                 Const.ParamsNames.COURSE_ID, "idOfTypicalCourse1"
         };
-        
+
         verifyOnlyInstructorsOfTheSameCourseCanAccess(params);
         verifyUnaccessibleWithoutModifyCoursePrivilege(params);
     }
-    
+
     @Test
-    public void testExecuteAndPostProcess() throws Exception{
-        //TODO: find a way to test status message from session
-        InstructorAttributes instructor1ofCourse1 =
-                dataBundle.instructors.get("instructor1OfCourse1");
-        
+    public void testExecuteAndPostProcess() throws Exception {
+        InstructorAttributes instructor1ofCourse1 = dataBundle.instructors.get("instructor1OfCourse1");
+
         ______TS("Not enough parameters");
-        
+
         gaeSimulation.loginAsInstructor(instructor1ofCourse1.googleId);
         verifyAssumptionFailure();
-        //TODO make sure IFAA does assertNotNull for required parameters then uncomment
-        //verifyAssumptionFailure(Const.ParamsNames.COURSE_ID, instructor1ofCourse1.courseId);
-        
+
+        // This is commented out as InstructorFeedbackAddAction already assertNotNull using Assumption
+        // verifyAssumptionFailure(Const.ParamsNames.COURSE_ID, instructor1ofCourse1.courseId);
+
         ______TS("Typical case");
-        
+
         FeedbackSessionAttributes session1 = dataBundle.feedbackSessions.get("session1InCourse1");
-        FeedbackQuestionAttributes question1 = FeedbackQuestionsLogic.inst().getFeedbackQuestion(session1.feedbackSessionName, session1.courseId, 1);
-        FeedbackQuestionAttributes question2 = FeedbackQuestionsLogic.inst().getFeedbackQuestion(session1.feedbackSessionName, session1.courseId, 2);
-        
+        FeedbackQuestionAttributes question1 = FeedbackQuestionsLogic.
+                                               inst().
```

Comment:
Ok, how about for unary and binary operators in that case, before or after?

And hmm, how about for JavaScript files, follow Google style and dot behind, or override and follow Teammates dot next line?

Great that styles are being ironed out:p was never sure which to follow.

Sent from my iPhone

> On 21 May 2015, at 9:02 am, Damith C. Rajapakse <notifications@github.com> wrote:
> 
> In src/test/java/teammates/test/cases/ui/InstructorFeedbackQuestionCopyActionTest.java:
> 
> >          FeedbackSessionAttributes session1 = dataBundle.feedbackSessions.get("session1InCourse1");
> > -        FeedbackQuestionAttributes question1 = FeedbackQuestionsLogic.inst().getFeedbackQuestion(session1.feedbackSessionName, session1.courseId, 1);
> > -        FeedbackQuestionAttributes question2 = FeedbackQuestionsLogic.inst().getFeedbackQuestion(session1.feedbackSessionName, session1.courseId, 2);
> > -        
> > +        FeedbackQuestionAttributes question1 = FeedbackQuestionsLogic.
> > +                                               inst().
> I'll update the guidelines. Even in the example, the dot operator appears in the next line :-p
> —
> Reply to this email directly or view it on GitHub.
>